examples/session_info/session_info.view.lkml

```diff
@@ -70,28 +70,28 @@ view: session {
   }
     
   dimension: java_runtime {
-    description: "Java Runtime"
+    description: "Java_runtime"
```

Comment:
The previous version looked nicer. Is that change intentional?

Reply:
No this wasn't intentional, I'll fix it.